phoenicis-library/src/main/java/org/phoenicis/library/ShortcutReader.java

```diff
@@ -0,0 +1,41 @@
+package org.phoenicis.library;
+
+import org.phoenicis.library.dto.ShortcutDTO;
+
+import java.util.List;
+
+/**
+ * Interface for all shortcut reader implementations
+ */
+public interface ShortcutReader {
+    /**
+     * Sets the shortcut
+     *
+     * @param shortcut The shortcut
+     */
+    void of(ShortcutDTO shortcut);
```

Comment:
I'm still thinking whether it is better to add the functionality of this method to the constructor. 

Changing the constructor to have the signature:
```java
public ShortcutReader(ShortcutDTO shortcut)
```

has the benefit that the object usage is cleaner. In addition it has the disadvantage that we can't add the constructor requirement to the interface.

Reply:
If it is part of the constructor, it cannot be defined in the interface, right?

Reply:
Exactly